### PR TITLE
Go back to using AppVeyor

### PR DESCRIPTION
**Pull request by oxinabox**
Revert "Merge pull request #237 from JuliaTime/cv/drop-appveyor"
This reverts commit 01254a6d501062f2aaf140c485b8181f0c4ea37e.

Because of https://github.com/JuliaTime/TimeZones.jl/pull/251#issuecomment-600360127
> > Travis CI windows is failing to build Plots because FFMPEG
> 
> More precisely, it's Travis fault that has an incomplete Windows installation, see [JuliaIO/FFMPEG.jl#14](https://github.com/JuliaIO/FFMPEG.jl/issues/14) and references therein

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -86,3 +87,19 @@ after_success:
...
+    - <<: *doc_test
+      os: linux
+      julia: nightly
+    - <<: *doc_test
+      os: osx
+      julia: 1.0
+    - <<: *doc_test
+      os: osx
+      julia: nightly
+    - <<: *doc_deploy
```
Comment (omus) [inline .travis.yml]: Looks like this shouldn't have been added back in

File: .travis.yml [hunk 2]
```diff
@@ -25,12 +25,13 @@ stages:
...
   - osx
-  - windows
+  - linux
 julia:
-  - 1.0  # LTS
-  - 1.3  # Latest release
+  - 1.0
+  - 1.1
+  - 1.2
+  - 1.3
```
Comment (omus) [inline .travis.yml]: Should be left as:
```
- 1.0  # LTS
- 1.3  # Latest release
```

File: appveyor.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+environment:
+  JULIA_TZ_VERSION: "2016j"
+
+  matrix:
+  - julia_version: 1.0
+  - julia_version: 1.1
+  - julia_version: 1.2
```
Comment (omus) [inline appveyor.yml]:
```suggestion
```

File: appveyor.yml [hunk 4]
```diff
@@ -0,0 +1,33 @@
+environment:
+  JULIA_TZ_VERSION: "2016j"
+
+  matrix:
+  - julia_version: 1.0
+  - julia_version: 1.1
```
Comment (omus) [inline appveyor.yml]:
```suggestion
```

File: appveyor.yml [hunk 5]
```diff
@@ -0,0 +1,33 @@
+environment:
+  JULIA_TZ_VERSION: "2016j"
+
+  matrix:
+  - julia_version: 1.0
```
Comment (omus) [inline appveyor.yml]:
```suggestion
  - julia_version: 1.0  # LTS
```

File: appveyor.yml [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+  - julia_version: 1.1
+  - julia_version: 1.2
+  - julia_version: 1.3
```
Comment (omus) [inline appveyor.yml]:
```suggestion
  - julia_version: 1.3  # Latest release
```

File: appveyor.yml [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+  - julia_version: latest
+
+matrix:
+  allow_failures:
+    # https://github.com/JuliaTime/TimeZones.jl/issues/218
+    - julia_version: latest
+
+platform:
+  - x86 # 32-bit
```
Comment (omus) [inline appveyor.yml]: We'll do the 32-bit testing on Travis since that's still enabled
```suggestion
```

File: appveyor.yml [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+  - julia_version: 1.0
+  - julia_version: 1.1
+  - julia_version: 1.2
+  - julia_version: 1.3
+  - julia_version: latest
+
+matrix:
+  allow_failures:
+    # https://github.com/JuliaTime/TimeZones.jl/issues/218
+    - julia_version: latest
```
Comment (omus) [inline appveyor.yml]: Should be removed as well

File: appveyor.yml [hunk 9]
```diff
@@ -0,0 +1,33 @@
+environment:
+  JULIA_TZ_VERSION: "2016j"
+
+  matrix:
+  - julia_version: 1.0
+  - julia_version: 1.1
+  - julia_version: 1.2
+  - julia_version: 1.3
+  - julia_version: latest
```
Comment (omus) [inline appveyor.yml]: This apparently works. Let's just update to that right away
```suggestion
  - julia_version: nightly
```

**Comment by omus** [comment]
Hmmm, I think the `appveyor.yml` file needs to be present in this repo before Appveyor will trigger? I'm rather unsure

File: .travis.yml [hunk 10]
```diff
@@ -25,12 +25,13 @@ stages:
 # Note: At the moment `codecov: true` and `coveralls: true` are not working.
 language: julia
 os:
-  - linux
   - osx
-  - windows
+  - linux
```
Comment (omus) [inline .travis.yml]:
```suggestion
  - linux
  # - windows  # Unable to use Windows on Travis due to: https://github.com/JuliaTime/TimeZones.jl/pull/251#issuecomment-600360127
```

File: .travis.yml (hunk 2 above)
Comment (oxinabox) [reply .travis.yml]: oops that was not ment to change

File: appveyor.yml (hunk 6 above)
Comment (oxinabox) [reply appveyor.yml]: A.V. Actually lets us specify that as `1`
https://github.com/JuliaCI/Appveyor.jl

File: .travis.yml [hunk 11]
```diff
@@ -26,11 +26,11 @@ stages:
 language: julia
 os:
   - linux
+  # - windows  # Unable to use Windows on Travis due to: https://github.com/JuliaTime/TimeZones.jl/pull/251#issuecomment-600360127
   - osx
-  - windows
 julia:
   - 1.0  # LTS
-  - 1.3  # Latest release
+  - 1.4  # Latest release
```
Comment (omus) [inline .travis.yml]: Travis CI also supports `1` for the latest release. That is still using 1.3 at the moment though: https://github.com/travis-ci/travis-build/blob/master/lib/travis/build/script/julia.rb#L185

**Comment by omus** [comment]
I wish AppVeyor was running again as part of this PR. I guess I'll just have to take a leap of faith

**Comment by oxinabox** [comment]
@omus  I am unsure what things are remaining for me to do in this PR?

File: .travis.yml (hunk 11 above)
Comment (omus) [reply .travis.yml]:
```suggestion
  - 1    # Latest release
```

File: appveyor.yml [hunk 12]
```diff
@@ -0,0 +1,25 @@
+environment:
+  JULIA_TZ_VERSION: "2016j"
+
+  matrix:
+  - julia_version: 1.0  # LTS (1.0.x)
+  - julia_version: 1  # Latest release (1.x.y)
```
Comment (omus) [inline appveyor.yml]:
```suggestion
  - julia_version: 1    # Latest release (1.x.y)
```

File: .travis.yml [hunk 13]
```diff
@@ -26,11 +26,11 @@ stages:
 language: julia
 os:
   - linux
+  # - windows  # Unable to use Windows on Travis due to: https://github.com/JuliaTime/TimeZones.jl/pull/251#issuecomment-600360127
   - osx
-  - windows
 julia:
   - 1.0  # LTS
```
Comment (omus) [inline .travis.yml]:
```suggestion
  - 1.0  # LTS (1.0.x)
```

**Comment by omus** [comment]
>  I am unsure what things are remaining for me to do in this PR?

The last thing was changing `1.4` in Travis to `1`. I've also updated the comments to make them consistent but I would have merged this without that. After that I'll merge this and deal with the CI if it's still not working afterwards.

**Comment by omus** [comment]
#251 found a work around which no longer required us to re-enable Appveyor.

**Pull request closed without merging**